library/src/main/java/com/kennyc/view/MultiStateView.java

```diff
@@ -326,18 +352,21 @@ public void setViewForState(View view, @ViewState int state, boolean switchToSta
             case VIEW_STATE_LOADING:
                 if (mLoadingView != null) removeView(mLoadingView);
                 mLoadingView = view;
+                mLoadingView.setTag(R.id.tag_multistateview, TAG_LOADING);
```

Comment:
Besides comparing the tags in `isValidContentView`, does setting the tags for the view accomplish anything else?